site/docs/education/tutorials/postman-tutorial/overview.md

```diff
@@ -45,24 +45,77 @@ postman files later, you can just create a new environment again to work with
 Select the workspace menu at the top of the Postman interface, and create a new one
 called 'Egeria dojo':
 
-![Creating a new Environment 1](postman-workspace1.png)
-![Creating a new Environment 2](postman-workspace2.png)
-![Creating a new Environment 3](postman-workspace3.png)
+![Creating a new Environment 1](postman-create-workspace1.png)
+![Creating a new Environment 2](postman-create-workspace2.png)
+![Creating a new Environment 3](postman-workspace-created.png)
 
 ## Importing Egeria Postman data
 
 We now will
-* Select 'Import' and select 'GitHub' as the source
-* Authenticate as required
-* Select the "main" branch from the odpi/egeria repository on GitHub
-* Import all the Postman data found
-
-![Import Begin](postman-import-begin.png)
-![Select import source](postman-import-coderepo.png)
-![Auth with GitHub](postman-import-gotogh.png)
-![Select egeria repo](postman-import-ghrepo.png)
-![Select what to import](postman-import-select.png)
-![After Import](postman-import-after.png)
+
+* Individually import the Egeria Postman collections required for this dojo
+* Select 'Import' and select 'Link' as the source
+* Paste the URL found below for the `Egeria-admin-services-operational` collection
+
+    ```
+    https://raw.githubusercontent.com/odpi/egeria/40ec581a8dbf82eeb7e96e2412099358193e8fc1/open-metadata-implementation/admin-services/Egeria-admin-services-operational.postman_collection.json
+    ```
+
+* Confirm the correct collection will be imported
+* Confirm the collection was imported successfully
```

Comment:
how should the user do this?

Reply:
I've given more detailed instructions, and included the screenshot in a collapsible section.